.gitignore

```diff
@@ -22,5 +22,4 @@ nb*.xml
 **/*.iml
 
 
-
-**/.flattened-pom.xml
```

Comment:
Unnecessary change.
Since `.flattened-pom.xml` is already tracked only (re-)adding to gitignore won't work. It must be explicitly deleted.

For reference see:
- https://git-scm.com/docs/gitignore#_description
- https://git-scm.com/docs/gitignore#_notes
- https://git-scm.com/docs/git-rm#Documentation/git-rm.txt---cached

Reply:
[90fb16d](https://github.com/BlackBeltTechnology/karaf-jasypt-support/pull/2/commits/90fb16dae3be9705ff56e77a2b4bc60309c32233)